docs/slack.rst

```diff
@@ -55,6 +54,13 @@ Other useful topical channels
 * **#community-help-wanted** - Got an open-source project that needs docs contributors? Looking for open-source communities who will mentor you while you contribute to their docs? This channel is for you! Only for volunteer/unpaid work with open-source projects; companies and paid job postings should continue to use #job-posts-only. 
 * **#wtd-weps** - Ask questions about the Write the Docs Enhancement Proposal (WEP) system and brainstorm on pre-proposal ideas to help improve our community processes and governance. 
 
+Channels that serve specific communities & their allies
+-------------------------------------------------------
+Allies are welcome as guests in these channels. Please be considerate.
+
+* **#bipoc** - Social and discussion space for Write the Docs' :doc:`/bipoc` community.
```

Comment:
@plaindocs Here I'm trying to link to the `bipoc.rst` doc .. but the doc doesn't exist in this PR. Is that a problem? 🤔

Reply:
@eirinikos  What I would usually do is change the smaller PR to target the branch of the of the larger PR, so in this case merge, `add-slack-channgels` into `mjang-add-bipoc`, and merge so that all links resolve, then double check you're happy with this one.